migration_const.py

```diff
@@ -1,4 +1,5 @@
 # CHANGE ME
 REPOSITORY_PATH = "<PROJECT_PATH>"
 DATA_PATH = REPOSITORY_PATH + "data/"
-ICON_PATH = REPOSITORY_PATH + "icon/"
\ No newline at end of file
+ICON_PATH = REPOSITORY_PATH + "icon/"
+MAPPING_PATH = REPOSITORY_PATH + "files/"
```

Comment:
name `files` could be appropriate for any file, please change it to something more specific e.g., `temp-files`

Reply:
It has already been changed